DylibBinder/DBElement.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace DylibBinder {
+	// TODO Add Elements to enums
+	public class DBElement {
+		public DBElement (string name, string intValue)
+		{
+			Name = name;
+			IntValue = intValue;
+		}
+
+		public string Name { get; }
+		public string IntValue { get; }
+	}
+
+	public class DBElements {
+		public DBElements ()
+		{
+		}
+
+		public List<DBElement> Elements { get; } = new List<DBElement> ();
+	}
```

Comment:
Same

Reply:
This one is also just a placeholder until I can determine if the elements are present somewhere.